### PR TITLE
rename 'clusters' in NewClusterModal

### PR DESCRIPTION
**Pull request by panentheos**
This is a subset of the rename effort.

### CODE REVIEW

File: src/components/NewClusterModal.js [hunk 1]
```diff
@@ -449,35 +449,35 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
...
 
-  getImageUrl(clusterDetails) {
-    return _.find(({ imageType }) => _.includes(imageType, ['Jupyter', 'RStudio']), clusterDetails?.runtimeImages)?.imageUrl
+  getImageUrl(runtimeDetails) {
+    return _.find(({ imageType }) => _.includes(imageType, ['Jupyter', 'RStudio']), runtimeDetails?.runtimeImages)?.imageUrl
   }
 
   componentDidMount = _.flow(
-    withErrorReporting('Error loading cluster'),
+    withErrorReporting('Error loading environment'),
```
Comment (s-rubenstein) [inline src/components/NewClusterModal.js]: Nit: I would maybe say `cloud environment` since that's what we call it to the user.

**Review by s-rubenstein — COMMENTED** [review]
https://github.com/DataBiosphere/terra-ui/pull/2259/files#diff-9fc1399eaeddca57bf342d5f239c71faR120 we should rename the component, probably. And also ideally rename the file.

File: src/components/NewClusterModal.js (hunk 1 above)
Comment (panentheos) [reply src/components/NewClusterModal.js]: Done.

**Review by s-rubenstein — APPROVED** [review]
LGTM

**Pull request merged**